modules/widgets/wordpress-post-widget.php

```diff
@@ -926,7 +926,12 @@ public function form( $instance ) {
 				<?php _e( "Enter a WordPress.com or Jetpack WordPress site URL.", 'jetpack' ); ?>
 			</i>
 			<?php
-			if ( empty( $url ) ) {
+			/**
+			 * Show an error if the URL field was left empty.
+			 *
+			 * The error is shown only when the widget was already saved.
+			 */
+			if ( empty( $url ) && ! preg_match( '/__i__|%i%/', $this->id ) ) {
```

Comment:
I am not sure `$this->updated` is the most optimal solution here. It is fired only when a change occurs, i.e. after saving, but it is not valid when the user reloads the widget edit screen. This results in no error shown when the user leaves/reloads the widget edit area.